src/main/java/core/basesyntax/MainApp.java

```diff
@@ -12,5 +12,19 @@
  * - Отправить все машини на работу и остановить работу всех машин.
  */
 public class MainApp {
+    public static void main(String[] args) {
+        Machine track = new Track();
+        track.doWork();
+        track.stopWork();
 
+        Machine bulldozer = new Bulldozer();
+        bulldozer.doWork();
+        bulldozer.stopWork();
+
+        Machine excavator = new Excavator();
+        excavator.doWork();
+        excavator.stopWork();
+    }
 }
+
```

Comment:
let's remove redundant lines at the end of the file

Reply:
ready